server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java

```diff
@@ -38,19 +37,24 @@
   private static final long serialVersionUID = 1L;
 
   private final String tableId;
+  private final String namespaceId;
```

Comment:
I'm surprised if inserting a new field in-between other fields actually works. I would have expected the order of the existing fields being disrupted to have an effect on serialization. This should be tested.